docs/source/tutorial/report.rst

```diff
@@ -0,0 +1,165 @@
+    ...
+    >>> observation
+    {'my_observer:/x': 1}
+
+When a value is passed to the ``reporter``, an object called ``observer`` can be
+optionally attached. In this case, the name of the ``observer`` is added as the
+prefix of the value name. The ``observer`` name should be registered beforehand.
+Using ``reporter.scope``, you can select which ``observation`` to save the
+observed values.
+
+There are also a global API :func:`chainer.report`, which reports observed values
+with the current reporter object. In this case, `current` means which ``with``
+statement scope the current code line is in. This function calls the
+:func:`Reporter.report()` method of the current reporter. 
+
+.. doctest::
+
+    >>> observation = {}
+    >>> with reporter.scope(observation):
+    ...     report({'x': 1}, observer)
+    ...
+    >>> observation
+    {'my_observer:/x': 1}
+
+Use report in Chain or Link
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The most important application of :class:`~chainer.Reporter` is to report
+observed values from each :class:`~chainer.Link` or :class:`~chainer.Chain`
+in the training and validation procedures.
+
+But, how to report the observed values from each link or chain? Shold we
+prepare the :class:`~chainer.Reporter`? No, you only need to call
+:func:`report` in chain or link,
+because :class:`~chainer.training.Trainer` and some extensions prepare their own
+:class:`~chainer.Reporter` object with the hierarchy of the target link registered
+as observers. We can use :func:`report` function inside any links and chains to
+report the observed values (e.g., training loss, accuracy, activation statistics, etc.).
+
+See the following example:
+
+.. doctest::
+
+    >>> class Classifier(Chain):
+    ...     def __init__(self, predictor):
+    ...         super(Classifier, self).__init__()
+    ...         with self.init_scope():
+    ...             self.predictor = predictor
+    ...
+    ...     def __call__(self, x, t):
+    ...         y = self.predictor(x)
+    ...         loss = F.softmax_cross_entropy(y, t)
+    ...         accuracy = F.accuracy(y, t)
+    ...         report({'loss': loss, 'accuracy': accuracy}, self)
+    ...         return loss
+    ...
+
+If the link is named ``'main'`` in the hierarchy (which is the default
+name of the target link in the :class:`~chainer.training.StandardUpdater`),
+these reported values are named ``'main/loss'`` and ``'main/accuracy'``.
+If these values are reported inside the :class:`~chainer.training.extensions.Evaluator`
+extension, ``'validation/'`` is added at the head of the link name, thus
+the item names are changed to ``'validation/main/loss'`` and ``'validation/main/accuracy'``
+(``'validation'`` is the default name of the Evaluator extension).
+
+Naming rule for the reported values
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+So, you have known almost everything about :class:`~chainer.Reporter`.
+However, one content is remained. It is what is the naming rule for the reported values,
```

Comment:
suggested wording change:
So, you know almost everything about :class:`~chainer.Reporter`.
 However, there is one more thing.